src/app.ts

```diff
@@ -1,9 +1,11 @@
 import express from "express";
 import userRouter from "./routers/userRouter";
+import groupRouter from "./routers/groupRouter";
 
 const app = express();
 app.use(express.json());
-app.use("/user", userRouter);
+app.use("/users", userRouter);
```

Comment:
suggest using common files with routes like this ```app.use(routes);``` . If we have 100 routes, what will happen?